examples/seq2seq/seq2seq.py

```diff
@@ -168,6 +169,8 @@ def main():
                         help='Resume the training from snapshot')
     parser.add_argument('--unit', '-u', type=int, default=1024,
                         help='Number of units')
+    parser.add_argument('--input', '-i', type=str, default="wmt",
+                        help="Input directory")
```

Comment:
Could you sort arguments? It looks good that `--out` and `--input` are adjacent anyway.